rulings/rulings.yaml

```diff
@@ -4292,7 +4290,6 @@
 200314|Dancin' Dana:
   - '[MERGED] Is Anarch. If the base version was the one in play, it changes sect as it is merged. [ANK 20180524]'
 200316|Danielle Diron:
-  - Keeps her title and votes if she changes sects. [LSJ 20010714]
```

Comment:
that doesn't work, you need to remove the key too since it's now empty. Otherwise, you'll get a yaml syntax error here